app/layout/projects/new/component.tsx

```diff
@@ -0,0 +1,241 @@
+                      <Field id="description" {...fprops}>
+                        <Label theme="dark" className="mb-3 uppercase">Description</Label>
+                        <Textarea rows={4} placeholder="Write your project description..." />
+                      </Field>
+                    )}
+                  </FieldRFF>
+                </div>
+
+                <h2 className="mt-12 text-white font-heading">
+                  Do you have a planning region shapefile?
+                </h2>
+
+                {/* PLANNING AREA */}
+                <div className="flex items-center justify-between mt-6">
+                  <div className="flex items-center">
+                    <h5 className="text-white uppercase text-xxs">Planning area</h5>
+                    <button
+                      className="w-5 h-5 ml-2"
+                      type="button"
+                      onClick={() => console.info('Planning Area info button click')}
+                    >
+                      <Icon icon={INFO_SVG} />
+                    </button>
+                  </div>
+                  <div className="flex">
+                    <Button
+                      className="w-20 h-6 mr-4"
+                      size="xs"
+                      theme={!hasPlanningArea ? 'primary-white' : 'secondary'}
+                      onClick={() => setHasPlanningArea(false)}
+                    >
+                      No
+                    </Button>
+                    <Button
+                      className="w-20 h-6"
+                      size="xs"
+                      theme={hasPlanningArea ? 'primary-white' : 'secondary'}
+                      onClick={() => setHasPlanningArea(true)}
+                    >
+                      Yes
+                    </Button>
+                  </div>
+                </div>
+
+                {!hasPlanningArea && <PlanningUnitGrid unit={null} />}
+
+                {/* BUTTON BAR */}
+                <div className="flex mt-8">
+                  <Button
+                    theme="secondary"
+                    size="xl"
+                    onClick={handleCancel}
+                  >
+                    Cancel
+                  </Button>
+                  <Button
+                    className="ml-6"
+                    theme="primary"
+                    size="xl"
+                    type="submit"
+                    disabled
+                  >
+                    Save
+                  </Button>
+                </div>
+              </form>
+            )}
+          </FormRFF>
+        </div>
+        <div className="relative flex items-center justify-center w-1/2 h-full text-white">
+          <Map
+            className=""
+            width="100%"
+            height="100%"
+            bounds={bounds}
+            minZoom={minZoom}
+            maxZoom={maxZoom}
+            viewport={viewport}
+            mapboxApiAccessToken={process.env.STORYBOOK_MAPBOX_API_TOKEN}
```

Comment:
The map is not going to work in the app because you didn't send the .env variable for that.
```suggestion
            mapboxApiAccessToken={process.env.NEXT_PUBLIC_MAPBOX_API_TOKEN}
```